src/test/java/org/folio/rest/impl/PatronBlockLimitsAPITest.java

```diff
@@ -1,10 +1,10 @@
 package org.folio.rest.impl;
 
-import static org.apache.http.HttpStatus.SC_CREATED;
-import static org.apache.http.HttpStatus.SC_UNPROCESSABLE_ENTITY;
+import static org.apache.http.HttpStatus.*;
```

Comment:
We're using a code style scheme where `Class count to use import with '*'` is `100`, so there probably should be separate imports.